infogami/infobase/server.py

```diff
@@ -208,7 +208,7 @@ def GET(self, sitename):
         i = input("key", revision=None, expand=False)
         site = get_site(sitename)
         if not site:
-            raise RuntimeError("get_site({}) failed".format(sitename))
+            logger.info("get_site({}) failed".format(sitename))
```

Comment:
This should log the exception as an exception, without halting execution.

```suggestion
            logger.exception("get_site({}) failed".format(sitename))
```